packages/shared.net.platformapi/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@osdk/shared.net.platformapi",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/palantir/foundry-platform-typescript.git"
+  },
+  "exports": {
+    ".": {
+      "browser": "./build/browser/index.js",
+      "import": "./build/esm/index.js"
+    },
+    "./*": {
+      "browser": "./build/browser/public/*.js",
+      "import": "./build/esm/public/*.js"
+    }
+  },
+  "scripts": {
+    "check-attw": "monorepo.tool.attw esm",
+    "check-spelling": "cspell --quiet .",
+    "clean": "rm -rf lib dist types build tsconfig.tsbuildinfo",
+    "fix-lint": "eslint . --fix && dprint fmt --config $(find-up dprint.json)",
+    "lint": "eslint . && dprint check  --config $(find-up dprint.json)",
+    "transpile": "monorepo.tool.transpile"
+  },
+  "dependencies": {
+    "@osdk/shared.client": "^1.0.1",
+    "@osdk/shared.client2": "^1.0.0",
+    "@osdk/shared.net.errors": "^2.0.1"
+  },
+  "devDependencies": {
+    "@osdk/monorepo.api-extractor": "workspace:~",
+    "@osdk/monorepo.tsconfig": "workspace:~",
+    "@osdk/monorepo.tsup": "workspace:~",
+    "typescript": "^5.5.4"
```

Comment:
Same here, I think we should be enforcing `~5.5.4`, but current MRL setup wanted ^. Can FLUP fix as well